flang/test/Lower/intrinsic-procedures.f90

```diff
@@ -805,6 +805,32 @@ subroutine repeat_test(c, n)
   return
 end subroutine
 
+! RESHAPE
+! CHECK-LABEL: reshape_test
+! CHECK-SAME: %[[arg0:.*]]: !fir.box<!fir.array<?x?xi32>>, %[[arg1:.*]]: !fir.box<!fir.array<?x?x?xi32>>, %[[arg2:.*]]: !fir.box<!fir.array<?x?x?xi32>>, %[[arg3:.*]]: !fir.ref<!fir.array<2xi32>>, %[[arg4:.*]]: !fir.ref<!fir.array<2xi32>>)
```

Comment:
This line is getting long... maybe split it with more `CHECK-SAME` lines?

Reply:
For some reason I was getting errors when I put each arg on its own line. If I keep arg1 and arg2 on its own line,  I can break up the others. I don't  know why that makes a difference, but it will be an improvement.

Reply:
`arg1` and `arg2` have the same types, so if split on its own line `%[[arg1:.*]] : !fir.box<!fir.array<?x?x?xi32>>,` will be very happy to match `arg1` with `arg1 : : !fir.box<!fir.array<?x?x?xi32>>, arg2` instead of simply `arg1` because both string match the regex, and the longer match is kept, which is not the intended match.
So what you did makes sense to me.